pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/controller/LoginController.java

```diff
@@ -63,6 +70,12 @@
 	@Autowired
 	private Environment environment;
 
+	@Value("${mosip.kernel.otp.expiry-time}")
+	private int optExpiryTime;
```

Comment:
spelling mistake? it should be otpExpiryTime not optExpiryTime

Reply:
will correct the spelling